vision/unit_tests/test_client.py

```diff
@@ -116,6 +116,24 @@ def test_face_detection_from_content(self):
                          image_request['image']['content'])
         self.assertEqual(5, image_request['features'][0]['maxResults'])
 
+    def test_face_detection_from_content_no_results(self):
+        RETURNED = {
+            'responses': [{}]
+        }
+        credentials = _Credentials()
+        client = self._make_one(project=PROJECT, credentials=credentials)
+        client._connection = _Connection(RETURNED)
+
+        image = client.image(content=IMAGE_CONTENT)
+        faces = image.detect_faces(limit=5)
+        self.assertEqual(faces, [])
+        self.assertEqual(len(faces), 0)
+        image_request = client._connection._requested[0]['data']['requests'][0]
```

Comment:
I'm not entirely sure what you mean here?

I'm just trying to access the request that was made to `Connection` and verify that the parameters that I set were actually passed to that level.

I've been cringing, looking at these tests and I want  to clean them up and speech with what I've learned since starting.